README.md

```diff
@@ -106,6 +106,29 @@ projects.  For this feature to work, you need to obtain a Google API key, and st
 it in `web/clientId.txt`.  If you don't do this, the sign-in and save features will
 not function correctly, but the rest of the site will be usable.
 
+To get a Google API key for your local CodeWorld installation, running at
+`http://localhost:8080`, follow these step:
+
+1. Go to [Google API Console](https://console.cloud.google.com/apis/)
+2. Click the project drop-down
+3. Click "+" on right-hand side to create a new project
+4. Under "New Project"
+    1. Enter _CodeWorld_ in "Project name"
+    2. Click "Create"
+5. From project drop-down, ensure that "CodeWorld" is selected
+6. Click on "Credentials" on the right-hand side under "API"
+7. Click "Create credentials" and select "OAuth client ID"
+8. Click "Configure consent screen"
+9. Enter _CodeWorld_ under "Product name shown to users"
+10. Click "Save"
+11. Under "Application type" select "Web application".
+12. Enter _CodeWorld_ under "Name"
+13. Enter _http://localhost:8080_ for both "Authorized JavaScript origins" and "Authorized redirect URIs"
```

Comment:
This only works when the host is accessed as localhost.  Perhaps it would be better to explain that you must add any protocol, host, and port combinations that will be used to access the site, and that localhost:8080 is the right choice for local testing.